ui/qt-lib/src/Ariadne/UI/Qt/Face.hs

```diff
@@ -176,6 +176,8 @@ type UiWalletTree = Tree UiWalletTreeItem
 -- to be an issue, we may consider changing it.
 type TreePath = [Word]
 
+type NonEmptyPath = NonEmpty Word
```

Comment:
`TreePath` is also used for `wtsPath` and that probably can be empty.

`uadiPath` in `UiAddressInfo` should be non-empty too, but in fact it's requirement is stronger -- it should always be `[a, b]` -- account and address.

So maybe instead of storing this paths we store explicit indices? One for account and two for address.

Reply:
Probably the fact that `uadiPath` has exactly 2 indices is currency-specific and in other currencies it can be different, so I'd make it `NonEmptyPath`.

Reply:
Okay. @goodsoft your word?

Reply:
Won't break Disciplina, thus fine for now.

Reply:
So let's make it `NonEmptyPath` and I'll approve.